apl-core/src/test/java/com/apollocurrency/aplwallet/apl/core/db/EntityDbTableTest.java

```diff
@@ -392,146 +394,6 @@ public void testGetManyOnConnectionWithoutCache() {
         );
     }
 
-    @Test
```

Comment:
Too many tests were deleted here, is there a reason for that? Need to replace it or fix to force them to pass successfully or, considering abstract hierarchy, develop special test case for test impl of entityDbTable, such test impl will let us an ability to break tight coupling between this abstract tests and other entities tests.

Reply:
I deleted some methods that were used only in tests, so I deleted redundant tests as well. No need to develop something else.